docs/metrics.md

```diff
@@ -78,15 +79,19 @@ overlay.error.read                       | meter     | error while receiving a m
 overlay.error.write                      | meter     | error while sending a message
 overlay.fetch.txset                      | timer     | time to complete fetching of a txset
 overlay.fetch.qset                       | timer     | time to complete fetching of a qset
+overlay.flood.advertised                 | meter     | transactions advertised through pull mode
+overlay.flood.demanded                   | meter     | transactions demanded through pull mode
+overlay.flood.fulfilled                  | meter     | demanded transactions fulfilled through pull mode
+overlay.flood.unfulfilled-banned         | meter     | transactions we failed to fulfilled since they are banned
+overlay.flood.unfulfilled-unknown        | meter     | transactions we failed to fulfilled since they are unknown
 overlay.flood.broadcast                  | meter     | message sent as broadcast per peer
 overlay.flood.duplicate_recv             | meter     | number of bytes of flooded messages that have already been received
 overlay.flood.unique_recv                | meter     | number of bytes of flooded messages that have not yet been received
 overlay.inbound.attempt                  | meter     | inbound connection attempted (accepted on socket)
 overlay.inbound.drop                     | meter     | inbound connection dropped
 overlay.inbound.establish                | meter     | inbound connection established (added to pending)
 overlay.inbound.reject                   | meter     | inbound connection rejected
-overlay.outbound-queue.scp               | timer     | time SCP traffic sits in flow-controlled queues
-overlay.outbound-queue.tx                | timer     | time tx traffic sits in flow-controlled queues
+overlay.outbound-queue.<X>               | timer     | time <X> traffic sits in flow-controlled queues
```

Comment:
missing overlay.demand.timeout

Reply:
I removed the `overlay.demand.timeout` metric